<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>KevaAddKeyDialog</class>
    <widget class="QDialog" name="KevaAddKeyDialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string notr="true">Add New Key-Value Pair</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QLabel" name="label_1">
                    <property name="text">
                        <string>Key</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QLineEdit" name="keyText">
                    <property name="toolTip">
                        <string>New value</string>
                    </property>
                </widget>
            </item>
            <item>
                <spacer name="horizontalSpacer">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                        <size>
                            <width>40</width>
                            <height>15</height>
                        </size>
                    </property>
                </spacer>
            </item>
            <item>
                <widget class="QLabel" name="label_2">
                    <property name="text">
                        <string>Value</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QTextEdit" name="valueText">
                    <property name="toolTip">
                        <string>New value</string>
                    </property>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="standardButtons">
                        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
</ui>
